### PR TITLE
Provides a new login-page for Tycho.

### PR DESCRIPTION
**Pull request by andyHa**
Fixes: SIRI-589, SIRI-590

### CODE REVIEW

File: src/main/resources/default/templates/biz/tycho/dashboard-news.html.pasta [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                                sirius.postJSON('/tycho/updates/markAsSeen',
+                                    {updateId: '@info.getGuid()'});
+                            });
+                        });
+                    </script>
+                </div>
+            </i:for>
+        </div>
+    </div>
+</i:if>
```
Comment (jakobvogel) [inline src/main/resources/default/templates/biz/tycho/dashboard-news.html.pasta:30]: LF@EOF

File: src/main/resources/default/templates/biz/tycho/page-footer-help.html.pasta [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                    const _footerContents = document.getElementById('tycho-page-footer-help');
+                    _footerContents.innerHTML = '@raw { @escapeJS(contents) }';
+                    sirius.executeEmbeddedScripts(_footerContents);
+
+                    document.getElementById('tycho-page-footer-nav').innerHTML = '@raw { @escapeJS(navbox) }';
+                });
+            });
+         </script>
+    </i:if>
+</i:if>
```
Comment (jakobvogel) [inline src/main/resources/default/templates/biz/tycho/page-footer-help.html.pasta:39]: LF@EOF

**Review by jakobvogel — APPROVED** [review]

File: src/main/resources/default/templates/biz/login.html.pasta [hunk 3]
```diff
@@ -2,12 +2,146 @@
...
+            }
+        });
+
+        function showForgetPasswordModal() {
+            const _emailField = document.getElementById('mailField');
+            const _userField = document.getElementById('user');
+
+            _emailField.value = _userField.value;
+
+            // This bootstrap call sill needs jQuery :(...
```
Comment (mkeckmkeck) [inline src/main/resources/default/templates/biz/login.html.pasta:118]:
```suggestion
            // This bootstrap call still needs jQuery :(...
```

File: src/main/resources/default/templates/biz/login.html.pasta [hunk 4]
```diff
@@ -2,12 +2,146 @@
...
+            $('#forgotPassword').modal('show');
+        }
+
+        function submitForgotPassword() {
+            const _emailField = document.getElementById('mailField');
+
+            // This bootstrap call sill needs jQuery :(...
```
Comment (mkeckmkeck) [inline src/main/resources/default/templates/biz/login.html.pasta:125]:
```suggestion
            // This bootstrap call still needs jQuery :(...
```

File: src/main/java/sirius/biz/tycho/TechStackInfo.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+import java.util.function.BiConsumer;
+
+/**
+ * Used to signal the usage of an open source technology.
+ * <p>
+ * This has to be {@link sirius.kernel.di.std.Register registered} and will then be picked up by the
+ * {@link DashboardController} to be shown on the login page.
+ */
+public interface TechStackInfo extends Priorized {
```
Comment (mkeckmkeck) [inline src/main/java/sirius/biz/tycho/TechStackInfo.java:21]: AutoRegister?

File: src/main/resources/default/templates/biz/tycho/page-footer-help.html.pasta [hunk 6]
```diff
@@ -0,0 +1,40 @@
+<i:arg type="String" name="class" default=""/>
+<i:local name="contents" value="renderToString('body')"/>
+<i:local name="navbox" value="renderToString('navbox')"/>
+
+<i:if test="isFilled(contents) || isFilled(navbox)">
+    <i:local name="contents" value="renderToString('body')"/>
+    <i:local name="navbox" value="renderToString('navbox')"/>
+    <i:if test="isFilled(contents) || isFilled(navbox)">
```
Comment (mkeckmkeck) [inline src/main/resources/default/templates/biz/tycho/page-footer-help.html.pasta:7]: duplicated

**Review by mkeckmkeck — APPROVED** [review]

File: src/main/resources/default/templates/biz/login.html.pasta (hunk 3 above)
Comment (ymo-sci) [reply src/main/resources/default/templates/biz/login.html.pasta:118]: is it available to comment out completely for the browser like <!--@ ?

Comment (sabieber) [inline src/main/resources/default/templates/biz/login.html.pasta:118]: Deshalb wär n Upgrade auf Bootstrap 5 gut gewesen :) Da ist das alles native

File: src/main/resources/default/extensions/tycho-page-menu/footer.html.pasta [hunk 7]
```diff
@@ -3,6 +3,16 @@
 
 <i:switch test="@point">
     <i:block name="footer-right-start">
+        <i:if test="!currentRenderContext().getRootContext().getTemplate().getName().endsWith('dashboard.html.pasta')">
```
Comment (sabieber) [inline src/main/resources/default/extensions/tycho-page-menu/footer.html.pasta:6]: Da gibts keinen besseren Check für? Vllt den Block im Dashboard Template überschreiben oder so?

**Review by sabieber — APPROVED** [review]

File: src/main/resources/default/extensions/tycho-page-menu/footer.html.pasta (hunk 7 above)
Comment (andyHa) [reply src/main/resources/default/extensions/tycho-page-menu/footer.html.pasta:6]: Would probably look as ugly as that. Sadly, we cannot simply extract this into a block, as we're already in an extension...

**Pull request merged**